src/com/tony/server/Main.java

```diff
@@ -8,6 +8,7 @@
 import com.tony.server.response.ImageContentResponse;
 import com.tony.server.response.DirectoryResponse;
 import com.tony.server.response.RedirectResponse;
+import com.tony.server.response.LogResponse;
```

Comment:
It seems like you have your imports in alphabetical order in most places. This one is out of order.